boards/arduino-mega2560/include/periph_conf.h

```diff
@@ -14,6 +14,7 @@
  * @brief       Peripheral MCU configuration for the Arduino Mega 2560 board
  *
  * @author      Hinnerk van Bruinehsen <h.v.bruinehsen@fu-berlin.de>
+ * @author      René Herthel <rene.herthel@haw-hamburg.de>
```

Comment:
If you like you can add the HAW copyright under FU Berlin. **If** you do so, don't forget it is 2015 currently